types/google-cloud__text-to-speech/index.d.ts

```diff
@@ -0,0 +1,123 @@
+// Type definitions for google-cloud__text-to-speech 0.5
+// Project: https://github.com/googleapis/nodejs-text-to-speech
+// Definitions by: Ben James <https://github.com/benhjames>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.8
+
+/// <reference types="node" />
+
+export type GoogleError = any;
+
+export type APICallback<T = any> = (
+    err: GoogleError | null,
+    response?: T
+) => void;
+
+export interface PromiseLike<T> extends Promise<T> {
+    /**
+     * Cancel the ongoing promise
+     */
+    cancel(): void;
+}
+
+export interface MethodOverload<T, R> {
+    (data: T, options?: CallOptions): PromiseLike<[R]>;
+    (data: T, options: CallOptions, callback: APICallback<R>): void;
+    (data: T, callback: APICallback<R>): void;
+}
+
+export interface CallOptions {
+    timeout?: number;
+    retry?: any;
+    autoPaginate?: boolean;
+    pageToken?: any;
+    isBundling: boolean;
+    longrunning?: any;
+    promise?: any;
+}
+
+export interface ClientOptionsCredentials {
+    client_email: string;
+    private_key: string;
+}
+
+export interface ClientOptions {
+    credentials?: ClientOptionsCredentials;
+    email?: string;
+    keyFilename?: string;
+    port?: number;
+    projectId?: string;
+    promise?: any;
+    servicePath?: string;
+}
+
+export interface ListVoicesRequest {
+    languageCode?: string;
+}
+
+export type ListVoicesOptions = CallOptions;
+
+export type SsmlVoiceGender =
+    | "SSML_VOICE_GENDER_UNSPECIFIED"
+    | "MALE"
+    | "FEMALE"
+    | "NEUTRAL";
+
+export interface Voice {
+    language_codes: string[];
+    name: string;
+    ssmlGender: SsmlVoiceGender;
+    naturalSampleRateHertz: number;
+}
+
+export type ListVoicesResponse = Voice[];
+
+export type SynthesisInput = { text: string } | { ssml: string };
+
+export interface VoiceSelectionParams {
+    languageCode?: string;
+    name?: string;
+    ssmlGender?: SsmlVoiceGender;
+}
+
+export type AudioEncoding =
+    | "AUDIO_ENCODING_UNSPECIFIED"
+    | "LINEAR16"
+    | "MP3"
+    | "OGG_OPUS";
+
+export interface AudioConfig {
+    audioEncoding: AudioEncoding;
+    effectsProfileId?: string[];
+    pitch?: number;
+    sampleRateHertz?: number;
+    speakingRate?: number;
+    volumeGainDb?: number;
+}
+
+export interface SynthesizeSpeechRequest {
+    input: SynthesisInput;
+    voice: VoiceSelectionParams;
+    audioConfig: AudioConfig;
+}
+
+export type SynthesizeSpeechOptions = CallOptions;
+
+export interface SynthesizeSpeechResponse {
+    audioContent: Buffer;
+}
+
+declare class TextToSpeechClient {
+    constructor(options?: ClientOptions);
+
+    listVoices: MethodOverload<ListVoicesRequest, ListVoicesResponse>;
+    synthesizeSpeech: MethodOverload<
+        SynthesizeSpeechRequest,
+        SynthesizeSpeechResponse
+    >;
+}
+
+declare const TextToSpeech: { TextToSpeechClient: typeof TextToSpeechClient };
+
+export default TextToSpeech;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
I can't seem to get any form of `export =` with namespaces to work (with varying problems, including [this one](https://github.com/Microsoft/dtslint/blob/master/docs/export-just-namespace.md)). If you could give any further guidance as to what else I need to do, that would be great! 👍 

(btw - [the package does adhere](https://github.com/googleapis/nodejs-text-to-speech/blob/master/src/index.js#L87) to the default export syntax, though it's faked using the older module system, does this make a difference?)

Thanks!

Reply:
Sorry about that. I missed seeing that it does have default property so this should be good. Thanks